src/patternfly/components/Card/card.scss

```diff
@@ -13,6 +14,12 @@
   background-color: var(--pf-c-card--BackgroundColor);
   box-shadow: var(--pf-c-card--BoxShadow);
 
+  &.pf-m-hover {
+    &:hover {
+      box-shadow: var(--pf-c-card--m-hover--hover--BoxShadow);
```

Comment:
Just want to verify this was an approved add from design? The shadow seems to big to me personally, like we need something between `md` and `lg` box shadows.

Reply:
And assuming so, we should add it to the documentation.

Reply:
and... potentially call it something else. we use `pf-m-hover` to force the hover state in components. Maybe `pf-m-hoverable`?

Reply:
- the shadow has been approved by design, @gdoyle1 want to make sure you're fine with the lg size box shadow.  I can test out the medium one also?
- agree with the hoverable name.